core/embed/prodtest/README.md

```diff
@@ -106,6 +106,48 @@ TOUCH 09
 OK 50 90
 ```
 
+
+### TOUCH_CUSTOM
+The `TOUCH_CUSTOM` command tests the functionality of the display's touch screen.
+It draws a filled rectangle on custom coordinates and waits for user interaction.
+
+The command requires five input parameters:
+
+* X position of the top-left corner of the rectangle
+* Y position of the top-left corner of the rectangle
+* Width of the rectangle
+* Height of the rectangle
+* The timeout in seconds
+
+If the display is not touched within the specified timeout, the command will return an `ERROR TIMEOUT`.
+
+The device report touch events, coordinates and timestamps (in ms), correctness of the touch point is not checked and is left to the test equipment.
+
+The test ends with first lift-up event.
+
+Example (to draw a 100x100 rectangle in the top-left (10,10) position and wait for 15 seconds for touch input):
+```
+TOUCH_CUSTOM 10 10 100 100 15
+TOUCH D 69 35 357300
+TOUCH U 69 35 357328
+OK
+```
+
+### TOUCH_IDLE
+The `TOUCH_IDLE` command tests the functionality of the display's touch screen.
```

Comment:
To be consistent with other commands, I would rather changed the command name to `TOUCH IDLE` (without the underscore). The same applies to `TOUCH_CUSTOM`.

Reply:
as mentioned in the PR description i went for the underscore to avoid conflicts with existing commands:
if you use `TOUCH CUSTOM `, it will also be recognized by` if(startswith("TOUCH ") `branch.

I could place the `TOUCH_CUSTOM` above the `TOUCH`, but still, older prodtest versions will recognize this new command as regular `TOUCH`. Using underscore prevents this.

If anything, i would go for using underscore everywhere, with space reserved as parameters delimiter. (or 0 delimiter for commands without params). This way we could avoid such conflicts systematically. But this would break compatibility with older test programs, so i am nor ready to do that yet.

Reply:
Hmm, I understand and agree. I don't like how it's done now either. We definitely should define this protocol/language precisely to avoid ambiguity in the future.